docs/versioned/admin/admin-overview.md

```diff
@@ -50,59 +112,40 @@ Knative configurations are performed by the following methods:
-
-Configurations for default settings:
-
-- [Broker defaults](../eventing/configuration/broker-configuration.md)
-- [ConfigMap defaults](../serving/configuration/config-defaults.md)
-- [Event source defaults](../eventing/configuration/sources-configuration.md)
-- [Channel defaults](../eventing/configuration/channel-configuration.md)
-- [Kafka channel defaults](../eventing/configuration/kafka-channel-configuration.md)
-- [Domain names](../serving/using-a-custom-domain.md)
-- [Ingress gateway replacement](../serving/setting-up-custom-ingress-gateway.md)
+    Some platform-wide settings can be managed declaratively using the Knative Operator, installed with the `kn` Knative CLI plugin. You can manage the operator without using the `kn` CLI. The `kn` CLI manages only operator installations.
 
-Configurations for new development:
+For more information, see [Installing CLI tools](../client/README.md)
 
-- [Deployment resources](../serving/configuration/deployment.md)
-- [Istio access to deployed services](../serving/istio-authorization.md)
-- [Namespace exclusion from webhooks](../serving/istio-authorization.md)
-
-Configurations for maintenance:
-
-- [Garbage collection](../serving/revisions/revision-admin-config-options.md)
-- [High availability](../serving/config-ha.md)
-- [Rollout duration for revisions](../serving/configuration/rolling-out-latest-revision-configmap.md)
-- [Autoscaling of Kafka features](../eventing/configuration/keda-configuration.md)
-
-Configurations for security encryptions:
-
-- [cert-manager](../serving/encryption/configure-certmanager-integration.md)
-- [External domains](../serving/encryption/external-domain-tls.md)
-- [Local domains](../serving/encryption/cluster-local-domain-tls.md)
-- [system-internal](../serving/encryption/system-internal-tls.md)
-
-Configurations for extensions:
-
-- [Kafka Broker features](../serving/encryption/system-internal-tls.md)
-- [Sugar Controller](../eventing/configuration/sugar-configuration.md)
-
-Configurations for flagging features:
-
-- [Serving features](../serving/configuration/feature-flags.md)
-- [Eventing features](../eventing/features/README.md)
-
-## Authorizations
-
-You can grant developers access to additional resources related to their namespace in other services, such as observability, logs, metrics, tracing, and dashboards.
+Knative uses Kubernetes YAML manifests to define and configure system components. These manifests include core resources, custom resource definitions (CRDs), and extensibility features. As with Kubernetes, these configuration resources are declarative and can be managed using the `kubectl` CLI tool or with continuous delivery tools.
 
-## Upgrades
+The following sections provide an overview of the current configurations procedures of interest to Administrators. You make these configurations using `kubectl`, with some procedures applying YAML files.
```

Comment:
```suggestion
The following sections provide an overview of the current configuration resources of interest to Administrators. You can edit these configurations using `kubectl`; Knative installs empty ConfigMaps with these names onto the cluster.
```

I'm not sure I'd call these "procedures" (or the act of editing them "configurations").